app/models/feed.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+class Feed < ActiveRecord::Base
```

Comment:
Also, we probably want an ActiveModel callback to queue the feed caching job every time that a feed is created or updated.